src/libsync/vfs/cfapi/vfs_cfapi.cpp

```diff
@@ -157,9 +157,9 @@ Result<void, QString> VfsCfApi::convertToPlaceholder(const QString &filename, co
 
     const auto handle = cfapi::handleForPath(localPath);
```

Comment:
@mmdanggg2 It looks like `cfapi::findPlaceholderInfo` is also asking to replace its argument type from `FileHandle` to `QString`. Now, we have to open a `handle` here just to check if placeholder info exists and then, we are not reusing that handle so it is just a redundant variable here. Also, it would be more consistent to always expect that `findPlaceholderInfo`, `updatePlaceholderInfo`, `convertToPlaceholder`, and `createPlaceholderInfo` all accept the same argument type.

Reply:
I've changed over all the wrapper functions taking a handle to use QString path instead.